app/src/main/res/values/styles.xml

```diff
@@ -74,4 +74,16 @@
         <item name="android:focusable">true</item>
     </style>
 
+    <!--Redesign Login Screen-->
+    <style name="TextInputLayout.StrokeWhite" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense">
+        <item name="boxStrokeColor">@color/white</item>
+        <item name="boxStrokeWidth">@dimen/layout_padding_2dp</item>
+    </style>
+
+    <style name="LoginTheme" parent="AppTheme">
+        <item name="android:windowBackground">@drawable/login_bg</item>
```

Comment:
can we convert png to vector drawable and use, can we give it a try?

Reply:
I have tried it before but the conversion is not perfect. It would be very helpful if you could provide an SVG file or a vector drawable file of the background.

Reply:
Let's use icons, which are used across other Mifos Apps.